DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateCourses/DelegateCoursesViewModelTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Delegates.DelegateCourses
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Helpers;
+    using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateCourses;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using NUnit.Framework;
+
+    public class DelegateCoursesViewModelTests
```

Comment:
Some of these are really tests of logic in BasePaginatedViewModel. Can we move them into a separate test file for that class?
Obviously because it's an abstract class, we have to pick some subclass to test the logic, but if the tests are in a place dedicated to BasePaginatedViewModel, it means others are less likely to write duplicate tests with other subclasses.

Reply:
I think this sounds like a good idea. I believe CourseSetup was just the first proper implementation of the full BaseSearchableViewModel, so that's where the tests ended up going. I think renaming CourseSetupViewModelTests and moving it to an appropriate folder would suffice. Then this file can be deleted. 

I think some of the CourseSetupControllerTests also follow a similar pattern of testing how we take in different parameters for pagination, but those will have some tests specific to those pages (e.g., default filter). Might be worth having a look through some of those and seeing if they are generic enough to belong in their own file.